src/spring/azext_spring/app.py

```diff
@@ -309,6 +309,11 @@ def app_update(cmd, client, resource_group, service, name,
     if no_wait:
         return
     wait_till_end(cmd, *pollers)
+
+    for p in pollers:
+        if p.status() == 'Failed' and isinstance(p.result(), Exception):
+            raise p.result()
```

Comment:
```suggestion
    if any(p.status() == 'Failed' and isinstance(p.result(), Exception) for p in pollers):
        raise next(p.result() for p in pollers if p.status() == 'Failed' and isinstance(p.result(), Exception))
```

nit: could be written like this in python